Biochemistry/reactions.json

```diff
@@ -4798,7 +4798,7 @@
         "id": "rxn00214", 
         "is_obsolete": 0, 
         "is_transport": 0, 
-        "linked_reaction": "rxn33514;rxn33931;rxn38827;rxn38829", 
+        "linked_reaction": "rxn33514;rxn33931;rxn37863;rxn38827;rxn38829", 
```

Comment:
I think in JSON these should be arrays not ; delimited strings

Reply:
Ah yes, actually that can be applied to any set of delimited strings, and particularly the stoichiometry field itself, I'd like to expand the library so that when it prints the JSON it's more structured.

Making that step will be necessary to move towards an entirely JSON-based approach, but a lot of the code I just wrote is based on parsing these strings, so I'd have to go back and re-write how the library interacts with the structured data.

Reply:
And when we do, we may want to call it linked_reactions to match the tense with the type. Just something to consider as we go forward

Reply:
Sure, you cool with merge?